pkg/sql/logictest/testdata/logic_test/alter_column_type

```diff
@@ -0,0 +1,65 @@
+# LogicTest: default parallel-stmts distsql distsql-metadata
```

Comment:
I want a test of normal stuff that is hard coded to something we know. Can you add this test:

```
statement ok
CREATE TABLE t (s STRING, sl STRING(5), t TIME, ts TIMESTAMP)

statement ok
INSERT INTO t VALUES ('some string', 'short', TIME '05:04:00', now())

statement ok
ALTER TABLE t ALTER s TYPE BYTES, ALTER sl TYPE STRING(6), ALTER t TYPE TIMETZ, ALTER ts TYPE TIMESTAMPTZ)

query TT
SHOW CREATE TABLE t
----
<whatever>

query TTTT
SELECT * FROM t
----
<whatever>
```